useful_extras/create_comparison.py

```diff
@@ -152,7 +172,7 @@ def recreate_single_figure(
                 else:
                     (mpl_line,) = ax.plot(centers, heights, label=name)
 
-                ax.scatter(additional_x, additional_y, c=mpl_line.get_color())
+                ax.scatter(additional_x[above_below], additional_y[above_below], c=mpl_line.get_color())
```

Comment:
I am not explicitly checking whether `len(additional_x) > 0` because for unyt arrays, `[]==[]` gives `[]` instead of `True`. Hence, if `above_below = []` and `additional_x = []`, then `additional_x[above_below]` will not cause an error.